.htaccess

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Keep this updated to point to the current release.
+RedirectMatch "/(javadoc|pydoc)/current" "/$1/2.6.0"
```

Comment:
Is there something added to the release guide to update this?

Reply:
would it be easy to make this redirect subpages also since we're adding this? then folks could use /current/ in links to specific javadoc/pydoc pages.  /current/* -->  /2.6.0/*

if it's not a simple change, I can file a JIRA request for the future. it's P2 for docs on the Beam site due to the site latest version variable that's updated every release, but this is a big pain point for non-Beam docs that point to specific Beam javadoc/pydoc as every link must be updated for each release.

Reply:
re: @alanmyrvold on release docs: @udim has a PR out for that; I'll work with him to make sure this gets added.

re: @melap on redirect subpages: yes, actually this is the default behavior! Documentation [here](https://httpd.apache.org/docs/current/mod/mod_alias.html#redirect).

Reply:
AWESOME!